.github/FUNDING.yml

```diff
@@ -1 +1 @@
-github: [OskarStark, VincentLanglet, jordisala1991, greg0ire]
```

Comment:
I think FUNDING.yml are limited to 4 people.
That's why we have GithubSponsorsExtension for others repos.

For dev-kit, we follow the list https://github.com/sonata-project/dev-kit/graphs/contributors manually

Reply:
This change was reverted. Thank you!